### PR TITLE
fix: updated error messages

### PR DESCRIPTION
**Pull request by sachins-geekyants**
Issue : https://github.com/atsign-foundation/at_widgets/issues/702

**- What I did**
- Updated error messages for following:
 **AtOnboardingResponseStatus.serverNotReached** : `Your atSign and the server is unreachable. Please try again or contact support@atsign.com`

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart [hunk 1]
```diff
@@ -267,7 +267,7 @@ class _AtOnboardingActivateScreenState
       } else if (e == AtOnboardingResponseStatus.serverNotReached) {
         await _showAlertDialog(
           e,
-          title: AtOnboardingLocalizations.current.error_server_not_found,
+          title: AtOnboardingLocalizations.current.msg_atSign_unreachable,
```
Comment (gkc) [inline packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart:270]: Nice - thank you for updating the other alert dialogs also, not just the one mentioned in the issue

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart [hunk 2]
```diff
@@ -267,7 +267,7 @@ class _AtOnboardingActivateScreenState
...
         );
       } else if (e == AtOnboardingResponseStatus.timeOut) {
         await _showAlertDialog(
```
Comment (gkc) [inline packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart:273]: Is title_session_expired the right message to show here if there was a timeout?

**Review by gkc — DISMISSED** [review]
LGTM but I had one question

File: packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart (hunk 2 above)
Comment (sachins-geekyants) [reply packages/at_onboarding_flutter/lib/screen/at_onboarding_activate_screen.dart:273]: Thanks for pointing it, fixed.

**Review by gkc — APPROVED** [review]
LGTM

**Comment by gkc** [comment]
Resolved by https://github.com/atsign-foundation/at_widgets/pull/723

**Pull request merged**